src/App.vue

```diff
@@ -148,7 +174,7 @@ class App extends Vue {
             designer: 'Francis',
         },
         {
-            label: 'Holiday Card',
+            label: 'Winter Holiday Card',
```

Comment:
If you try [google](https://www.google.com/search?tbm=isch&source=hp&biw=1280&bih=654&ei=8mN4XpyRBYGl-Qayt5bwCw&q=holiday+card&oq=test&gs_l=img.3..0l10.856.1249..1322...0.0..0.56.153.3......0....1..gws-wiz-img.kyxiWTf7cec&ved=0ahUKEwic0rKYiLDoAhWBUt4KHbKbBb4Q4dUDCAY&uact=5) - you'll see that "Holiday Card" will return winter only.
In my opinion, the problem is that we drop Christmas because it's a huge topic in the U.S.A. - while it is not in Europe and many other places. Don't know about middle east though. 

Hmmm, so while writing this, I actually came to agree, putting "Winter" in front will help people to understand what it is. :)